co2eq/purchase/footprints.yml

```diff
@@ -2405,6 +2405,170 @@ _children:
           - PURCHASE_CATEGORY_HEALTHCARE_PHARMARCY
         source: 'http://www.balticproject.org/en/calculator-page'
         unit: EUR
+  Heating:
+    _children:
+      Coal boiler:
```

Comment:
is this the purchase of a coal boiler?
I don't think it should be in purchase.yml if that's the case

Reply:
"if that's not the case", correct? 
where would you move it to?

Reply:
yes you're right, i forgot the "not" :D (I've edited my previous comment)
I think I would create an energyfootprint.yml in co2eq/energy/